azure-setup-guide/terraform/main.tf

```diff
@@ -0,0 +1,86 @@
+# Resource group setup
+resource "azurerm_resource_group" "xtdb_infra" {
+  name     = "xtdb-resources"
```

Comment:
is this 'aligning equals' a terraform standard? if not, I generally prefer not to, because it means changing the indentation of the whole paragraph if you either add a longer one or delete the longest one

Reply:
It would certainly seem to be - format wise I've used `terraform fmt` (see docs [**here**](https://developer.hashicorp.com/terraform/cli/commands/fmt)) to do all of the file formatting and was responsible for the aligning of the equals.

From their docs it is "used to rewrite Terraform configuration files to a canonical format and style.", so I'm thinking we stick with it.